src/com/lftechnology/java_training/jan12/WordManipulator.java

```diff
@@ -0,0 +1,85 @@
+package com.lftechnology.java_training.jan12;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Takes the words in its argument list and prints out any duplicate words, the number of distinct words, and a list of the words with
+ * duplicates eliminated. Hint: Try with HashSet and List implementation.
+ * 
+ * @author Utsab Neupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class WordManipulator {
+	private static final String[] WORDS_LIST = { "hello", "world", "how", "are", "you", "hello", "what", "are", "you", "doing" };
+	private static final Logger LOGGER = Logger.getLogger(WordManipulator.class.getName());
+
+	public static void main(String[] args) {
+		List<String> listOfWords = new ArrayList<>(Arrays.asList(WORDS_LIST));
+		LOGGER.log(Level.INFO, "list of words: {0}", listOfWords);
+		Set<String> dupEliminatedWords = new HashSet<>();
+		Set<String> dupWords = new HashSet<>();
+		Set<String> distinctWords = new HashSet<>();
+		// get duplicate eliminated words
+		dupEliminatedWords = getDupEliminatedWords();
+		// get duplicate words
+		dupWords = getDupWords();
+		// get distinctWords
+		distinctWords = getDistinctWords();
```

Comment:
And you can replace this with `dupEliminatedWords.removeAll(dupWords);` to get non-repeated words only.